test/EmailAddressTest.php

```diff
@@ -741,8 +741,8 @@ public function testUseMxCheckBasicValid()
         ];
 
         if (extension_loaded('intl')) {
-            $emailAddresses[] = 'иван@письмо.рф';
```

Comment:
The current validator only accepts RFC 5321 email addresses (atext = ALPHA / DIGIT / "!" / "#" / "$" / "%" / "&" / "'" / "*" / "+" / "-" / "/" / "=" / "?" / "^" / "_" / "`" / "{" / "|" / "}" / "~"). Because of the wrong use of IDNA for local parts, validate() returned false positives for local parts with unicode characters. When we implement [RFC 6532 #3.2](https://tools.ietf.org/html/rfc6532#section-3.2) the user should be able to choose if he wants to validate against RFC 5321 or RFC 6532 because SMTPUTF8 is not yet widely supported and just an extension